bin/travis/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+bloopDir="bloop-server"
+
+function bloop_version {
+  cat build.sbt | grep "val bloop" | sed 's|[^0-9.]||g'
+}
+
+[ -d "$bloopDir" ] && rm -rf "$bloopDir"
+mkdir "$bloopDir"
+
+bloopURL="https://github.com/scalacenter/bloop/releases/download/v$(bloop_version)/install.py"
+
+wget "$bloopURL" && python install.py --dest $bloopDir
+
+$bloopDir/bloop ng-stop
+$bloopDir/bloop server &>$bloopDir/bloop.log &
```

Comment:
You can replace all of this script by coursier launching bloopgun-core, the invocation will start bloop in the background of it isn’t running and then connect to the server and run the nailgun command you specified

Reply:
Do you have any manual for it at hand? I can't find any.

Reply:
`coursier launch ch.epfl.scala:bloopgun-core_2.12:1.3.4 -- <command>` will work